signingscript/test/integration/test_autograph.py

```diff
@@ -217,25 +207,50 @@ def _extract_compress_type_per_filename(path):
     makedirs(copied_file_folder)
     shutil.copy(original_file_path, copied_file_folder)
 
-    zip_infos_before_signature = _extract_compress_type_per_filename(os.path.join(copied_file_folder, file_name))
-
     context.config['signing_server_config'] = _write_server_config(tmpdir)
     context.task = _craft_task([file_name], signing_format='autograph_focus')
 
     keystore_path = os.path.join(tmpdir, 'keystore')
     certificate_path = os.path.join(TEST_DATA_DIR, 'autograph_apk.pub')
     certificate_alias = 'autograph_focus'
-    _instanciate_keystore(keystore_path, certificate_path, certificate_alias)
+    _instantiate_keystore(keystore_path, certificate_path, certificate_alias)
 
     await async_main(context)
 
-    signed_path = os.path.join(tmpdir, 'artifact', file_name)
-    assert _verify_apk_signature(keystore_path, signed_path, certificate_alias)
+def _extract_apk_signature_algorithm(apk_path):
+    cmd = [
+        _get_java_path('jarsigner'), '-verify', '-verbose',
+        apk_path
+    ]
+    log.info("running {}".format(cmd))
+    command = subprocess.run(
+        cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, universal_newlines=True
+    )
+
+    assert command.returncode == 0
+
+    algorithm_line = [line for line in command.stdout.split('\n') if 'Signature algorithm:' in line][0]
+    algorithm = algorithm_line.strip().split(' ')[2]
+    return algorithm[:-1]  # remove comma at end of algorithm
+
+
+@pytest.mark.asyncio
+@skip_when_no_autograph_server
+@pytest.mark.parametrize('format,expected_algorithm', ((
+    'autograph_apk_fennec_sha1', 'SHA1withSHA1withRSA',
+), (
+    'autograph_apk_fennec', 'SHA256withSHA256withRSA',
```

Comment:
Add TODO instruction to say we should verify the digest algorithm once SHA1 is not automatically added anymore

Reply:
Is this the same todo as from [here](https://github.com/mozilla-releng/signingscript/pull/96#discussion_r243585396)?

Reply:
Nope. These are 2 different TODOs. The other comment is for the signature algorithm. Please let me know if you need more details